pkg/util/exec/exec.go

```diff
@@ -265,7 +265,8 @@ func runCommandWithOutput(cmd *exec.Cmd, combinedOutput bool) (string, error) {
 	} else {
 		output, err = cmd.Output()
 		if err != nil {
-			out = fmt.Sprintf("%s. %s", string(output), string(err.(*exec.ExitError).Stderr))
+			out = strings.TrimSpace(fmt.Sprintf("%s. %s", string(output), string(err.(*exec.ExitError).Stderr)))
```

Comment:
How about assigning to output, then the rest of the code can continue as expected below, right?
```suggestion
			output = fmt.Sprintf("%s. %s", string(output), string(err.(*exec.ExitError).Stderr))
```